org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/format/IFormatterParticipant.java

```diff
@@ -48,6 +53,22 @@ default boolean formatAttributeValue(String name, String valueWithoutQuote, Char
 		return false;
 	}
 
+	/**
+	 * Format the given attribute value.
+	 * 
+	 * @param formatterDocument the formatter document
+	 * @param indentLevel       the current indent level of the attribute
+	 * @param formattingOptions the formatting options that contain the settings
+	 * @param attr              the attribute node
+	 * @param edits             the text edit list
+	 * @return true if the given attribute can be formatted and false otherwise.
+	 */
+	default boolean formatAttributeValueNew(XMLFormatterDocumentNew formatterDocument,

```

Comment:
I would set attr parameter as the first parameter because it is a formatter for attribute